<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1031</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGraphicsView" name="gv_input"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="gv_map"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="lv_layers"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGraphicsView" name="gv_deconv"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tw_weight"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tw_map"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAll"/>
   <addaction name="actionShowInput"/>
   <addaction name="actionShowLayerInfo"/>
   <addaction name="actionShowMap"/>
   <addaction name="actionShowWeight"/>
   <addaction name="actionShowDiff"/>
   <addaction name="actionShowDeconv"/>
   <addaction name="actionshowlayerLoss"/>
  </widget>
  <action name="actionAll">
   <property name="text">
    <string>GetAll</string>
   </property>
   <property name="toolTip">
    <string>GetAllInformation</string>
   </property>
  </action>
  <action name="actionShowDiff">
   <property name="text">
    <string>ShowDiff</string>
   </property>
  </action>
  <action name="actionShowWeight">
   <property name="text">
    <string>ShowWeight</string>
   </property>
  </action>
  <action name="actionShowMap">
   <property name="text">
    <string>ShowMap</string>
   </property>
  </action>
  <action name="actionShowLayerInfo">
   <property name="text">
    <string>ShowLayerInfo</string>
   </property>
  </action>
  <action name="actionShowDeconv">
   <property name="text">
    <string>ShowDeconv</string>
   </property>
  </action>
  <action name="actionShowInput">
   <property name="text">
    <string>ShowInput</string>
   </property>
  </action>
  <action name="actionshowlayerLoss">
   <property name="text">
    <string>showlayerLoss</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
